src/components/buttons/button-remove-image.jsx

```diff
@@ -0,0 +1,37 @@
+import ButtonCommand from '../base/button-command.js';
+import ButtonIcon from './button-icon.jsx';
+import ButtonStateClasses from '../base/button-state-classes.js';
+import React from 'react';
+
+/**
+ * The ButtonRemoveImage class removes an image using a CKEDITOR.command.
+ *
+ * @class ButtonRemoveImage
+ * @uses ButtonCommand
+ */
+class ButtonRemoveImage extends React.Component {
+	static defaultProps = {
+		command: 'removeImage',
+	};
+
+	static key = 'removeImage';
+
+	/**
+	 * @inheritDoc
+	 */
+	render() {
+		const cssClass = `ae-button ${this.getStateClasses()}`;
```

Comment:
Looking forward to getting away from this pattern (where we call a method that we don't define on the class instance but inherit magically due to our HOC's use of `extends`).